### DISCUSSION TITLE
404 File not found

### DISCUSSION THREAD

**Discussion by Anya556** [community/community#38814] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello!
I created repository and pushed proect, but cann't open link 
anya556.github.io.It has an error 404 File not found

**Comment by grahamzemel** [comment, marked as answer]
Make sure your filesystem has an accessible 'index.html' on your repository, and then make sure that you're using the correct branch, likely 'main' or 'master' in the Github Pages settings for your repo. 

Your repository appears to have an 'Index.html', not an 'index.html'. These paths are case sensitive, so you'll need to change it to all lowercase.

**Reply by Anya556** [reply]
Thanks)

**Comment by airtower-luna** [comment]
Your repository https://github.com/Anya556/anya556.github.io doesn't have an `index.html`, only `Index.html`. URLs are case sensitive. Rename the file and it should show up, like https://anya556.github.io/Index.html does now. :wink:

**Comment by Anya556** [comment]
Thank you! 😊

Чт, 10 нояб. 2022 г. в 22:33, Airtower ***@***.***>:
